Pages/test.cshtml.cs

```diff
@@ -31,7 +31,7 @@ public class testModel : PageModel
         public bool isSearchNeighborhood { get; set; }
         public bool isSearchZipCode { get; set; }
 
-        public string Url { get; set; }
+        public new string Url { get; set; }
```

Comment:
Added "new string" in place of "string", to help making it private in cs.html.cs

Reply:
Wow, another new one to me.  That's two in one review.

https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/new-modifier